src/content/insertable-streams/video/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<!--
+ *  Copyright (c) 2015 The WebRTC project authors. All Rights Reserved.
+ *
+ *  Use of this source code is governed by a BSD-style license
+ *  that can be found in the LICENSE file in the root of the source
+ *  tree.
+-->
+<html>
+<head>
+   <meta http-equiv="refresh"
+   content="0; url=//webrtc.github.io/samples/src/content/insertable-streams/video-processing/">
```

Comment:
So this means it will redirect instantly? Maybe let it take like 5 seconds to let the user understand that the page has moved?

Reply:
Should src/content/getusermedia/source/index.html also be changed?

FWIW, it seems people generally use 0 here, e.g. https://opensource.com/article/19/7/permanently-redirect-github-pages

Reply:
Ah okay if it's the way of doing things, LGTM.